e2e/tests/interchain_accounts/base_test.go

```diff
@@ -228,3 +238,182 @@ func (s *InterchainAccountsTestSuite) TestMsgSendTx_FailedTransfer_InsufficientF
 		})
 	})
 }
+
+func (s *InterchainAccountsTestSuite) TestMsgSubmitTx_SuccessfulTransfer_AfterReopeningICA() {
+	t := s.T()
+	ctx := context.TODO()
+
+	// setup relayers and connection-0 between two chains
+	// channel-0 is a transfer channel but it will not be used in this test case
+	relayer, _ := s.SetupChainsRelayerAndChannel(ctx)
+	chainA, chainB := s.GetChains()
+
+	// setup 2 accounts: controller account on chain A, a second chain B account.
+	// host account will be created when the ICA is registered
+	controllerAccount := s.CreateUserOnChainA(ctx, testvalues.StartingTokenAmount)
+	chainBAccount := s.CreateUserOnChainB(ctx, testvalues.StartingTokenAmount)
+
+	var (
+		portID      string
+		hostAccount string
+	)
+	initialChannelId := "channel-1"
+	channelIdAfterReopening := "channel-2"
```

Comment:
Good variable naming, you can also group these:
```suggestion
	var (
		portID      string
		hostAccount string
		
		initialChannelId = "channel-1"
	        channelIdAfterReopening = "channel-2"
	)
```

Commit the change locally, github doesn't do formatting

Can you rename `ChannelId` -> `ChannelID`?